test/topology/test_start_bootstrapped_with_invalid_seed.py

```diff
@@ -11,29 +11,42 @@
 
 from test.pylib.internal_types import IPAddress
 from test.pylib.manager_client import ManagerClient
-from test.pylib.util import wait_for_cql_and_get_hosts, wait_for_cql
+# from test.pylib.util import wait_for_cql_and_get_hosts, wait_for_cql
 
 logger = logging.getLogger(__name__)
 
 
 @pytest.mark.asyncio
-@pytest.mark.xfail(reason="issue #14945")
 async def test_start_bootstrapped_with_invalid_seed(manager: ManagerClient):
     """
-    Check that when bootstrapped, having a non-resolving DNS name in
-    the config, does prevent node restart.
     Issue https://github.com/scylladb/scylladb/issues/14945.
 
+    Check non-regression: try starting a non-bootstrapped node with an invalid seed.
+     1. Start 1 node with an invalid seed hostname.
+     2. Make sure starting the node fails qwith an error message.
+
+    Check that when bootstrapped, having a non-resolving DNS name in
+    the config, does prevent node restart.
      1. Start 3 nodes.
      2. Wait till all start and join the cluster.
      3. Stop one node & and start it with a non-existing name in the seeds list config.
      4. Make sure the node started successfully since it's already a cluster member.
-
     """
-    servers = await manager.servers_add(3)
 
-    s1 = servers[0]
+    s1 = await manager.server_add(start=False)
+
+    # Start the first node with an invalid seed and make sure it fails with an error message.
+    await manager.server_start(s1.server_id, seeds=[IPAddress("no_address")],
+                               expected_error="Bad configuration: invalid value in 'seeds'", wait_interval=20)
+
+    # Add 2 more nodes.
+    await manager.servers_add(2)
```

Comment:
Frankly I thought a cluster must contain more than one node. Removed that.

Reply:
In fact you are adding this to the test/topology/ suite, so the test already starts with a 3-node cluster, which is defined in test/topology/suite.yaml (`cluster` `initial_size` parameter).

So the first node you add  in the test is the 4th node in the cluster.

TBH we could move the test to test/topology_custom/ suite, because the test really needs only 1 node. But test/topology/ is also fine. There is work in progress by QA team to clean up this suite mess.